src/plugins/image/src/main/js/ui/Dialog.js

```diff
@@ -62,6 +62,9 @@ define(
 
         src = imgElm ? dom.getAttrib(imgElm, 'src') : '';
 
+        // Workaround to remove instance origin if it's been added, i.e. image has been repositioned
+        src = Utils.removeOriginFromSrc(src);
```

Comment:
Since the change is in `Dialog.js` does this mean that the user will need to open the insert/edit image dialog to fix the path after repositioning an image? What happens when user does not open the dialog? Should we execute this type of cleanup when user is submitting the content?